exercises/C-variables/exercise.js

```diff
@@ -1,3 +1,7 @@
 // Start by creating a variable `greeting`
 
+var greeting = "Hello World!"
```

Comment:
As per CYF guidelines we should refrain from using var when declaring a variable and instead use let and const.

Reply:
I've been guilty of this mistake all throughout my code:)